api/src/main/java/org/openmrs/module/pihmalawi/metadata/CommonMetadata.java

```diff
@@ -217,7 +217,11 @@ public EncounterType getAppointmentEncounterType() {
 			MasterCardConcepts.SIX_A,
 			MasterCardConcepts.THIRTEEN_A,
 			MasterCardConcepts.FOURTEEN_A,
-			MasterCardConcepts.FIFTEEN_A
+			MasterCardConcepts.FIFTEEN_A,
+			"cf770b14-4c9d-11ea-b77f-2e728ce88125",
```

Comment:
Hi @tlimbani , can you please define those UUIDs as constants in the MasterCardConcepts class? Similar to the variables above. It would be easier to read and understand the code. Thanks.

Reply:
Thanks for reviewing @cioan this makes sense. Will make the changes.

Reply:
@cioan made the changes and committed here https://github.com/PIH/openmrs-module-pihmalawi/pull/41/commits/7d077815b668c963aaae4a5abc1282a715981537

Reply:
Thanks @tlimbani . I will merge it. 
Sorry, I just noticed that there was another PR(#41) open. Should I merge that first? Should I merge them in order? Thanks!

Reply:
Thanks @cioan the previous pull had issues which I fix in the latest pull. I did not know that it was going to have bring 2 PRs if I used the same feature branch. 

But the older / earlier PR has one more change unrelated to this. But otherwise its okay to just merge the latest PR and I will redo the other modification (MLW-1125). Or if you have ideas let me know and I will change!

Reply:
Thanks @tlimbani ! I am merging this PR.